test_scripts/ATF_ShutDown_MASTER_RESET.lua

```diff
@@ -0,0 +1,118 @@
+Test = require('user_modules/dummy_connecttest')
```

Comment:
@LitvinenkoIra, please use template for atomic scripts: https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/develop/test_scripts/template_of_script.lua